### DISCUSSION TITLE
Say `users` not `items` in the user-directory UI.

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15597] [Feature Requests]
### Describe the Bug

It's a tiny thing, but seems more appropriate to say "users" in this module.

https://user-images.githubusercontent.com/62306630/190447618-a2d61a57-f67f-4fa2-8990-0be5fe48ea3d.mov

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
